### ISSUE TITLE
do you need nested contexts or other 1.1 features?

### ISSUE THREAD

**Issue by VladimirAlexiev** [w3c-ccg/traceability-vocab#291]
This is not an ISSUE just a placeholder to hold some info.

The Traceability effort mixes and matches from many places and is huge, so may well need to use JSONLD 1.1 features:
- nested contexts
- per-class context
- `@protect`
- ...

Scoped contexts allow you to define a term locally, so it
- resolves to a different prop, 
- applies different datatypes
- prepends a different namespace
- etc

https://github.com/w3c/vc-data-model/issues/843#issuecomment-1023167843 gives some examples from EPCIS.

**Comment by VladimirAlexiev** [comment]
@OR13 
>DID and VC contexts should not require 1.1 since they just define terms and don't bundle other ontologies.
But other contexts that have to bundle or integrate with many different ontologies have a good reason to consider 1.1 features?

Yes. In fact https://w3id.org/traceability/v1 (as linked in #285) uses nested per-type context, so it needs version 1.1 (as it declares).

**Comment by OR13** [comment]
yep, I am willing to accept that we may very well need `1.1` features here.

I am just always looking for ways to reduce the surface and increase the off the shelf tooling support, but I think our work is complex enough to justify the cost of `1.1`.
